olive/systems/azureml/aml_system.py

```diff
@@ -125,14 +126,70 @@ def __init__(
         self.env_vars = self._get_hf_token_env(self.azureml_client_config.keyvault_name) if self.hf_token else None
         self.temp_dirs = []
 
+    def submit_workflow(self, workflow_id: str):
+        logger.info("Submitting workflow %s to the AzureML system.", workflow_id)
+        ml_client = self.azureml_client_config.create_client()
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            workflow_pipeline = self._create_pipeline_for_workflow(workflow_id, tempdir)
+            self._run_workflow_job(
+                ml_client,
+                workflow_pipeline,
+                "olive-workflow",
+                tags={"Workflow": workflow_id},
+            )
+
+    def _create_pipeline_for_workflow(self, workflow_id: str, tmp_dir):
```

Comment:
I was thinking whether we should add this support as the local model and data could be large and uploading them would take a long time. @devang-ml what do you think?

Reply:
We should not limit the use case.

But, we should be smart because the data is usually coming from HF or other places anyway. In those cases, there is no need to download the data locally and upload it to AML system.